product-scenarios/9-access-delegation/9.1-access-delegation-with-end-user-authorization/9.1.1-authorization-code-grant-flow/src/test/java/org/wso2/identity/scenarios/sso/test/oauth/OAuthAuthorizationGrantTest.java

```diff
@@ -120,59 +138,168 @@ private void deleteApplication() throws IOException {
                      "Application deletion failed.");
     }
 
-    @Test(description = "9.1.1.1")
-    public void intiAuthorizeRequest() throws Exception {
+    @DataProvider(name = "provideAuthorizeRequestParams") public Object[][] provideAuthorizeRequestParams() {
+        return new Object[][] {
+
+                //test with mandatory valid request parameters.
+                { client, authzEndpoint, clientId, REDIRECT_URL, null, null, true, null, null },
```

Comment:
Can you double check the formatting here. Looks like we need to fix the indentation.